### DISCUSSION TITLE
TypeError

### DISCUSSION THREAD

**Discussion by BiffoQ** [deepanshs/mrinversion#72] [Q&A]
Hey!

I am trying to play around with the example in the following link: https://mrinversion.readthedocs.io/en/latest/galley_examples/relaxation/plot-7Cs-2Al-91Si.html#sphx-glr-galley-examples-relaxation-plot-7cs-2al-91si-py.

A type error (**TypeError: T2.__init__() got an unexpected keyword argument 'inverse_dimension'**) is encountered on the `relaxation.T2` reference. See code below:

```
relaxT2 = relaxation.T2(
    kernel_dimension=kernel_dimension,
    inverse_dimension=dict(
        count=32,
        minimum="1e-3 s",
        maximum="1e4 s",
        scale="log",
        label=r"log ($\lambda^{-1}$ / s)",
    ),
)
inverse_dimension = relaxT2.inverse_dimension
K = relaxT2.kernel(supersampling=20)
```
I have checked the relation.py script https://github.com/deepanshs/mrinversion/blob/master/mrinversion/kernel/relaxation.py and am still confused about this.

How can this be fixed, please?

Thanks.

**Comment by deepanshs** [comment]
Hi,

Yes, that error is confusing. I did a fresh installation and couldn't reproduce it.

Can you try installing the library directly from the git repo
`pip install git+https://github.com/deepanshs/mrinversion`

and try again.


If the problem still exists, can you provide following details on the installation:
- what OS are you working with (mac (intel or arm), windows, or linux)
- did you use pip for installation
- the version of numpy

Deepansh
